src/styles/stopwatch.scss

```diff
@@ -0,0 +1,47 @@
+.stopwatch {
+  height: $stopwatch-size;
+  width: $stopwatch-size;
+  border: $border-size dotted #000;
```

Comment:
The variable `$border-size` is used but not defined within this file. Ensure that `$border-size` is defined in the `variables` file or another imported file.